### ISSUE TITLE
Erasing dogma can't continue the game

### ISSUE THREAD

**Issue by comotur** [UnderscoreKilburn/repentance-issues#2036]
I was playing with bethany and used the eraser a lot, so when i fought dogma i had like 7 eraser wisps. I fought the first phase normally and when i killed the second phase the game didn't continue and i didn't know why. After some time I tried to spawn dogma with mods and I realised that he did the eraser effect on spawn so i think I just erased him accidentaly with an eraser wisp on the last hit and that stops the game from continuing since there was no way to respawn him.

Basically i was stuck at home forever with nowhere to go.

Video of what happened with the game: https://youtu.be/RAHm4jkllmk?si=DHMnuDgKi-yv7mi2

**Comment by epfly6** [comment]
Duplicate #365
